eidos/eidos_globals.cpp

```diff
@@ -1886,7 +1886,8 @@ unsigned int Eidos_PhysicalCoreCount()
 	size_t len;
 	unsigned int ncpu;
 	len = sizeof(ncpu);
-	sysctlbyname("hw.physicalcpu_max", &ncpu, &len, NULL, 0);
+    //Removed by Sudharshan for Linux compatibility
+	//sysctlbyname("hw.physicalcpu_max", &ncpu, &len, NULL, 0);
```

Comment:
I imagine this will generate a warning about ncpu being uninitialized, etc.?  Probably best to just comment out the entirely of Eidos_PhysicalCoreCount(), and its declaration in the header.  Thanks.